wallet/wallet-controller/src/read.rs

```diff
@@ -49,6 +52,8 @@ pub struct ReadOnlyController<'a, T> {
     account_index: U31,
 }
 
+type MapAddressWithUsage<T> = BTreeMap<ChildNumber, (Address<T>, bool)>;
```

Comment:
In such aliases, it would be nice to explain in a little comment above it what this really means. Like the bool at the end is ambiguous. I know what it's because I'm... me... but the reader won't generally understand.